Dockerfile

```diff
@@ -8,18 +8,13 @@ ENV ACS_ROOT=$ACS_PREFIX/acs
 
 ENV JAVA_HOME="/usr/java/default"
 
-# install deltarpm prior to installing everything else
-# it might save some time during downloading and installing the
-# dependencies below, but it is not urgently needed for ACS to work
-# c.f. https://www.cyberciti.biz/faq/delta-rpms-disabled-because-applydeltarpm-not-installed/
-RUN yum update -y && yum install -y deltarpm && \
```

Comment:
deltarpm .. was introduced by me some time ago .. I wanted to save some time during yum installs ... I think it never really saved time. 
In Centos8 `deltarpm` seems to be renamed or so ... so for simplicity I just removed it.